hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -470,6 +470,8 @@ public void close() throws IOException {
     // OM should be shutdown as the StateMachine has shutdown.
     LOG.info("StateMachine has shutdown. Shutdown OzoneManager if not " +
         "already shutdown.");
+    super.close();
+    ozoneManager.close();
```

Comment:
`OzoneManager.close` only calls `stop`, which `shutdown` already does, so this unnecessary.  I think it also prevents system exit, since `shutdown` does nothing if OM is already stopped.

```suggestion
```

Reply:
thanks for pointing out this , will fix it !